source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.h

```diff
@@ -133,6 +133,7 @@ class JsonTranscoderConfig : public Logger::Loggable<Logger::Id::config>,
   bool match_incoming_request_route_{false};
   bool ignore_unknown_query_parameters_{false};
   bool convert_grpc_status_{false};
+  bool stream_newline_delimited_{false};
```

Comment:
Maybe replace "print_options_" in line 131 with:
```
ResponseTranslateOptions  response_translate_options_;

Reply:
I wasn't sure if it was clearer to have this class match the shape of `proto` of the configuration or underlying implementation from the extension. In the `proto`, `stream_newline_delimited` is its own top-level option. It seems like a question of taste (since we can't replace `print_options` with `response_translate_options` in the top-level configuration + `proto` file breaking backwards-compatibiltiy), and I'm happy to go with either.

Reply:
I feel it is cleaner to put this new `stream_newline_delimited` inside `PrintOptions`.  It is part of print.